utils/prometheus/README.md

```diff
@@ -0,0 +1,555 @@
+# Substrate Prometheus Node Exporter
+![grants](./photo_2019-12-13_16-32-53.jpg)
+## Introduction
+
+Prometheus is one of the most widely used monitoring tool for managing high availability services supported by [Cloud Native Computing Foundation](https://www.cncf.io/). By providing Prometheus exporter in substrate, node operators can easily adopt widely used display/alert tool such as Grafana without seting-up/operating external Prometheus agent through RPC connections. Easy access to such monitoring tools will benefit parachain develepers/operators and validators to have much higher availability quality of their services.
+
+## List of Contents
+
+Hack Prometheus in Substrate
+ - Prometheus starter
```

Comment:
What is a `Prometheus starter`? Maybe a `Prometheus 101` or a `Prometheus primer`?

Reply:
complete